### PR TITLE
HIVE-24581: Remove AcidUtils call from OrcInputformat for non transactional tables

### PR DESCRIPTION
**Pull request by pvargacl**
### What changes were proposed in this pull request?

- Remove unneccessary AcidUtils.getAcidState call from OrcInputformat when the table is not transactional
- Remove redundant filesystem utility functions from AcidUtils to HdfsUtils

### Why are the changes needed?
Make the code more readable

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Current unit tests.

### CODE REVIEW

File: ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java [hunk 1]
```diff
@@ -1866,92 +1870,6 @@ private static boolean isDirUsable(Path child, long visibilityTxnId, List<Path>
...
-      }
-    } else {
-      List<FileStatus> children = HdfsUtils.listLocatedStatus(fs, dir, hiddenFileFilter);
-      for (FileStatus child : children) {
-        if (child.isDirectory()) {
-          if (recursive) {
-            originals.addAll(findOriginals(fs, child.getPath(), useFileIds, ignoreEmptyFiles, true));
-          }
-        } else {
-          if (!ignoreEmptyFiles || child.getLen() > 0) {
```
Comment (kuczoram) [inline ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java:1919]: As far as I see, the ignoreEmptyFiles parameter is removed and it is not present in the newly added methods. Is this parameter not used any more? (Just want to make sure that removing it won't have any side effect.)

**Review by kuczoram — COMMENTED** [review]
Thanks for the patch. It looks good to me, I just have one question.

File: ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java (hunk 1 above)
Comment (pvargacl) [reply ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java:1919]: The findoriginals was called with ignoreEmpty = true always, so i removed it. If I understand correctly this parameter could be removed from every other acidutils call. It was there to handle MR, which created empty files for every bucket. https://issues.apache.org/jira/browse/HIVE-13040?focusedCommentId=15159223&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15159223

Comment (kuczoram) [reply ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java:1919]: Oh, ok, I see. Thanks for the details.

**Review by kuczoram — APPROVED** [review]

**Pull request merged**